sdk/rust/oak_loader/src/main.rs

```diff
@@ -0,0 +1,86 @@
+//
+// Copyright 2020 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+//! An utility binary to run Oak Runtime.
+//!
+//! To invoke, run the following command from the root of the repository:
+//!
+//! ```
+//! cargo run --package=oak_loader -- --application=<APP_CONFIG_PATH>
+//! ```
+
+use clap::{App, Arg, ArgMatches};
+use oak_runtime::configure_and_run;
+use oak_runtime::proto::ApplicationConfiguration;
+use prost::Message;
+use std::fs::File;
+use std::io::Read;
+
+fn get_arguments() -> ArgMatches<'static> {
+    App::new("Oak Loader")
```

Comment:
FYI https://docs.rs/structopt/0.3.12/structopt/ seems to have a nicer API. See

https://github.com/project-oak/oak/blob/09c7d6d86dd3a6c3808f3157884dc702c5352a50/runner/src/internal.rs#L24-L32

Reply:
Done